docs/spec/supply/03_future_improvements.md

```diff
@@ -0,0 +1,8 @@
+# Future improvements
+
+The current supply module only keeps track of the `Total` supply of coins held in the network. This module might be updated in the future to a track all the [money supply](https://en.wikipedia.org/wiki/Money_supply) hold within a chain. That design would have to implement the different types of "**M**"s but applied to Cosmos' BPoS.
+
+Future improvements may also include other types of supply such as:
+
+* **Liquid Supply:** Supply of the network that is not locked. Coins locked can be delegated, undelegating or vesting (as part of a `VestingAccount`).
```

Comment:
IMHO we shouldn't make any of these assumptions about what kinds of supplies chains will have. It would be neat here if instead we stated that we'd like to figure out a way for applications to "register" their desired supplies and the ability to calculate them passively.

Reply:
I like that suggestion. We should change the `gov` spec future improvements in that case too